xdsl/interpreters/__init__.py

```diff
@@ -27,6 +29,7 @@ def register_implementations(
     ctx: MLContext,
     include_wgpu: bool = True,
```

Comment:
I'd rather add a flag here and make the client of this class choose to include onnx, like in the wgpu case